docs/source/conf.py

```diff
@@ -1,28 +1,25 @@
 # -*- coding: utf-8 -*-
 # alot documentation build configuration file
 
-import sys, os
+import sys
+import os
 
+from alot import __version__,__author__
```

Comment:
Will this find the correct files on disk? It should find the development code in ../../alot if we are in most cases (git checkout or tarball extract). But I assume it will find a installed version of alot in the system or user dirs first.